src/redux/thunks/updateFileEntry.ts

```diff
@@ -39,6 +39,7 @@ export const updateFileEntry = createAsyncThunk(
         if (getFileStats(filePath)?.isDirectory() === false) {
           fs.writeFileSync(filePath, payload.content);
           fileEntry.timestamp = getFileTimestamp(filePath);
+          fileEntry.text = payload.content;
```

Comment:
This might be outside the scope of the PR but we should probably rename `payload.content` to `payload.text` to avoid mistakes

Reply:
refactored this part